src/hw_isolation_record/entry.cpp

```diff
@@ -90,7 +91,11 @@ void Entry::resolveEntry(bool clearRecord)
             {
                 hw_isolation::utils::setEnabledProperty(
                     _bus, std::get<2>(assoc), true);
-                break;
+            }
+            if (std::get<0>(assoc) == "isolated_hw_errorlog")
```

Comment:
Did you test with the latest driver, particularly the PEL changes that prohibit PEL when associated with an HWIsolation entry?

I'm a bit surprised it's working without erasing the HWIsolation entry, as shown below (LN102), because the association between PEL and HWIsolation is only removed when the HWIsolation DBus entry object is deleted.

Reply:
Will try with latest and will make the change to delete it after the entry object is deleted.